pkg/lightning/backend/local.go

```diff
@@ -1830,34 +1882,38 @@ outside:
 		select {
 		case kvs, ok := <-w.kvsChan:
 			if !ok {
+				w.markCleanQueue()
```

Comment:
Since all kvs are consumed, the current state must be either 0 or 1, do we really need this action? I think it's redundant.